### PR TITLE
Added resources limits in the configuration of helm chart

### PR DESCRIPTION
**Pull request by rajneesh-ops**
**Description**

Added support to create Kubernetes deployment with a resource limit.

https://github.com/KohlsTechnology/eunomia/issues/205

**Type of change**

* Bugfix (non-breaking change which fixes an issue)

**Checklist**

- [X] Documentation updated

### CODE REVIEW

**Comment by rajneesh-ops** [comment]
eunomia

Above is the screenshot of the changes.

File: deploy/helm/eunomia-operator/values.yaml [hunk 1]
```diff
@@ -34,17 +34,17 @@ eunomia:
...
-      #   memory: 128Mi
-      # requests:
-      #   cpu: 100m
-      #   memory: 128Mi
+      limits:
+        cpu: 1000m
+        memory: 512Mi
+      requests:
+        cpu: 200m
+        memory: 256Mi
```
Comment (Smiley73) [inline deploy/helm/eunomia-operator/values.yaml]: Memory requests and limits must be the same.

Comment (seanmalloy) [reply deploy/helm/eunomia-operator/values.yaml]: For starters let's try this ...

```
limits:
  cpu: 200m
  memory: 256Mi
requests:
  cpu: 200m
  memory: 256Mi
```

**Comment by rajneesh-ops** [comment]
I have made the suggested changes.

**Pull request merged**